easybuild/easyconfigs/g/GULP/GULP-6.0-foss-2020a.eb

```
@@ -0,0 +1,51 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# Based on GULP-5.1-intel-2019a.eb
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'GULP'
+version = '6.0'
+
+homepage = 'https://gulp.curtin.edu.au/gulp/'
+description = """GULP is a program for performing a variety of types of simulation on materials
+ using boundary conditions of 0-D (molecules and clusters), 1-D (polymers), 2-D (surfaces, slabs
+ and grain boundaries), or 3-D (periodic solids)Band Unfolding code for Plane-wave based calculations"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {
+    'openmp': True,
+    'usempi': True,
+}
```

As far as I can tell openmp is not required here (this option makes it link e.g. fftw3_omp.so), at least I can't find anything in the gulp sources. Did I miss anything? If not please use the below, else put both on the same line (see the toolchain dict) for consistency with other ECs

```suggestion
toolchainopts = {'usempi': True}
```

---

I was actually wondering about the `openmp` as well but as my magic is not that advanced, I thought I leave it in there. This came from the `GULP-5.1-intel-2019a.eb` file which I used as a template. I have removed it.